### DISCUSSION TITLE
Copilot in Visual Studio blocks Watch and Conditional Breakpoint

### DISCUSSION THREAD

**Discussion by wjadevries** [community/community#32290] [Copilot Conversations]
After installing Copilot in Visual Studio 2022 v17.3.3 I cannot enter any text into a Watch or Conditional Breakpoint anymore.
The Copilot icon is shown in the textbox and text entry is blocked.

![image](https://user-images.githubusercontent.com/38012502/189101643-33d8eb64-3a33-4d7e-a8a5-91634d0a9009.png)

![image](https://user-images.githubusercontent.com/38012502/189101684-73b74705-6fc7-48b7-9ace-531974def132.png)

**Comment by coderdude87** [comment]
I can confirm that this is an issue.   Even though you can type in the condition, the characters typed are not displayed so you can not see what you are typing.

**Comment by iamcjwilliams** [comment]
Hello @wjadevries 

Please visit this project readme at https://github.com/github/copilot-vs#troubleshooting to troubleshoot your VS issue.

**Comment by tilmann1** [comment]
@wjadevries @coderdude87 Please try the just released Copilot 1.44.0.0 VS extension which contains various improvements that help address this issue on VS 17.3.x. Thanks!
